### ISSUE TITLE
Ability to configure dns.nodelocal setting

### ISSUE THREAD

**Issue by pbnj** [rancher/terraform-provider-rancher2#350]
This setting is exposed through Rancher's UI:
```
dns:
    linear_autoscaler_params: {}
    node_selector: null
    nodelocal: {}
    provider: coredns
    reversecidrs: null
    stubdomains: null
    update_strategy: {}
    upstreamnameservers: null
```

But, it does not appear to be supported by the terraform provider.

**Comment by rawmind0** [comment]
Not all the resources exposed by Rancher API are supported on the provider. `nodelocal` support is included on PR #354

**Comment by rawmind0** [comment]
PR #354 is already merged on master. It'll be included on next provider release. Please reopen issue if needed.
